.github/pull_request_template.md

```diff
@@ -0,0 +1,63 @@
+<!--
+Thanks for submitting a pull request! Here are some helpful tips:
+
+* Always create branches on and target the `develop` branch.
+* Run all the tests locally and ensure that they are passing.
+* Run `cargo fmt --all` to ensure that the code is formatted.
+* Small commits and contributions which attempt one single goal is preferable.
```

Comment:
It doesn't matter in our case, because it will be squashed

Reply:
I think you may have misread what I had wrote. I'm just saying that 1 PR should only accomplish 1 single goal. Don't accomplish multiple things in 1 PR.